@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -1042,8 +1037,8 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
 
         // Prevent that extruded buildings are completely flat (can introduce errors in normal
         // computation and extrusion).
-        if (height === floorHeight) {
-            height += MIN_BUILDING_HEIGHT;
+        if (Math.abs(height - floorHeight) < ExtrusionFeatureDefs.MIN_BUILDING_HEIGHT) {
+            height = ExtrusionFeatureDefs.MIN_BUILDING_HEIGHT;
```

Comment:
I guess it should rather be:
`height = floorHeight + ExtrusionFeatureDefs.MIN_BUILDING_HEIGHT`
or simply (without if):
'height = max(floorHeight + ExtrusionFeatureDefs.MIN_BUILDING_HEIGHT, height)'

Reply:
Fixed as suggested, thanks.